internal/exec/util/file.go

```diff
@@ -139,6 +139,14 @@ func RenderFile(l *log.Logger, c *resource.HttpClient, f types.File) *File {
 		tmp := int(gid)
 		f.Group.ID = &tmp
 	}
+	if f.User.ID == nil {
+		tmp := 0
```

Comment:
`internal/util`, not `internal/exec/util`. I'm pretty sure there are other places in the code which should use this new function instead of reinventing a solution.